sklearn/setup.py

```diff
@@ -39,6 +39,8 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('impute/tests')
     config.add_subpackage('inspection')
     config.add_subpackage('inspection/tests')
+    config.add_subpackage('inspection/_plot')
```

Comment:
I don't think this is necessary because I added a `setup.py` in the `inspection` module.